ui/src/app/views/workspace/component.ts

```diff
@@ -191,10 +195,38 @@ export class WorkspaceComponent implements OnInit {
   }
 
   openCluster(): void {
-    const url = 'https://leonardo.dsde-dev.broadinstitute.org/notebooks/'
+    const leoBaseUrl = 'https://leonardo.dsde-dev.broadinstitute.org';
```

Comment:
This needs to be configurable on a per-environment basis. (Add a TODO if you don't want to deal with that now.)

Reply:
Yup, I'll throw a TODO there.